public/components/FacetEntry.vue

```diff
@@ -77,9 +79,11 @@ export default Vue.extend({
 		});
 
 		this.facets.on('facet-histogram:rangechangeduser', (event: Event, key: string, value: any, facet: any) => {
+			const isNumber = !_.isNaN(_.toNumber(value.from.label[0]));
 			const range = {
-				from: _.toNumber(value.from.label[0]),
-				to: _.toNumber(value.to.label[0])
+				from: isNumber ? _.toNumber(value.from.label[0]) : Date.parse(value.from.label[0]) / 1000,
```

Comment:
I would suggest extracting the range creation out into a function since the logic is repeated in a few places.  Also make the `1000` a constant at the top of the file.

Reply:
For defining constants see https://github.com/vuejs/vue/issues/6004#issuecomment-312275817